apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -27,7 +28,24 @@ function SectionProgressSelector({
   sectionId,
   hasSeenProgressTableInvite,
 }) {
-  const [hasJustToggledViews, setHasJustToggledViews] = React.useState(false);
+  const [hasJustToggledViews, setHasJustToggledViews] = useState(false);
+
+  useEffect(() => {
+    const params = queryParams('view');
+    if (params === 'v2') {
+      setShowProgressTableV2(true);
```

Comment:
Do we also need to add anything from else from `onShowProgressTableV2Change` below?

Should we set the UserPreference or emit metrics when this happens?

Reply:
Ah, yeah, I do think we want to have a set UserPreferences...  I will add that.  The current metrics include a `Section New Progress Viewed` amplitude event.  @dju90 are there other metrics you want logged from this redirect from marketing or does marketing provide you all that data?

Reply:
Still need the UserPreference

Reply:
There's Google Analytics showing clicks on this marketing banner, so no other amplitude metrics needed at this time.

Reply:
> Still need the UserPreference

What's the status of this?

Reply:
See below for where the update UserPreferences is included.